nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_LOMASS/metadata.yaml

```
@@ -18,43 +18,39 @@ implemented_observables:
     description: Drell-Yan Mass Distribution
     label: ATLAS low-mass DY 2011
     units: ''
-  process_type: EWK_MLL
-  tables: []
-  npoints: []
+  process_type: DY_MLL
+  tables: [6]
+  npoints: [6]
   ndata: 6
   plotting:
-    kinematics_override: ewk_mll_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS low-mass DY 2011
     y_label: $d\sigma_{Z/\gamma^{*}}/dM_{ll}$ (fb)
-    plot_x: k2
+    plot_x: m_ll
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - m_ll
+  - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
-        units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
-    file: kinematics_M.yaml
+      m_ll:
+        description: squared mass
+        label: m_ll
```

```suggestion
        label: "$m_{ll}$'
```

But this is not "squared mass" ? (either the description or the variable are wrong?)

---

Hy @scarlehoff thanks for noticing this. The description is wrong so I changed it along with the label (latex compat).